File: assessment.js
var aantal_giraffen = parseInt(prompt('hoeveel giraffen heb je geteld'));
var aantal_struisvogels = parseInt(prompt('hoeveel struisvogels het je geteld'));
var aantal_zebras = parseInt(prompt('hoeveel zebras heb je geteld'));

function poten(giraffen, struisvogels, zebras){
    return giraffen * 4 + struisvogels * 2 + zebras * 4;
}
var oplossing = poten(aantal_giraffen,aantal_struisvogels,aantal_zebras);
document.getElementById('opdracht').innerHTML = oplossing

File: assessment.py
aantal_giraffen = int(input('hoeveel giraffen heb je geteld'))
aantal_struisvogels = int(input('hoeveel struisvogels het je geteld'))
aantal_zebras = int(input('hoeveel zebras heb je geteld'))
def poten(giraffen, struisvogels, zebras):
    return giraffen * 4 + struisvogels * 2 + zebras * 4 
print(poten(aantal_giraffen,aantal_struisvogels,aantal_zebras))
